torch_xla/runtime.py

```diff
@@ -70,6 +101,7 @@ def device_type() -> Optional[str]:
   return pjrt_device.split('_')[0] if pjrt_device else pjrt_device
 
 
+@run_once
 def using_pjrt() -> bool:
```

Comment:
Hah, this function also needs to get deprecated since I assume this is always `True`

Reply:
We still need to call `__maybe_select_default_device()`, this is the point why I call using_pjrt() only once.